glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
Oh yeah, above I was saying I thought it was supposed to be doing the dbus check_config call in rtslib, but I cannot find it. If it does not exists, I am fine with just saying delete all check_configs now and whoever has time first can fix the netlink interface so it works like how we all expected it to.  There would not be any type of regression then since it has always been busted.

If that is what you guys are saying then that I am ok with it.